src/site/checkstyle.xml

```diff
@@ -143,7 +146,7 @@
         <module name="IllegalInstantiation"/>
         <module name="InnerAssignment"/>
         <module name="MagicNumber">
-            <property name="ignoreNumbers" value="-1, 0, 1, 2, 3, 4, 5, 100"/>
```

Comment:
I need to have the 1 - 10 items from an array for the parser, It's pointless to initiate them on top, I could probably use an index though